### PR TITLE
Apply iau convention for position angle in sky ellipse

### PR DESCRIPTION
**Pull request by luca-giunti**
This PR addresses #2228, by applying as suggested the IAU convention for the rotation of the SkyEllipse.

I have a short [notebook](https://github.com/luca-giunti/share/blob/master/Untitled10.ipynb) to test the new behaviour.

### CODE REVIEW

**Comment by luca-giunti** [comment]
Thanks @lmohrmann, I adapted the test and changed the reference for the rotation. Updated [notebook](https://github.com/luca-giunti/share/blob/master/Untitled10.ipynb)

**Comment by cdeil** [comment]
Maybe make a https://astropy-regions.readthedocs.io/en/latest/api/regions.EllipseSkyRegion.html and overplot it on the model image. Is the rotation angle definition the same?

File: gammapy/image/models/core.py [hunk 1]
```diff
@@ -322,7 +322,7 @@ class SkyEllipse(SkySpatialModel):
         from gammapy.image.models.core import SkyEllipse
         from gammapy.maps import Map, WcsGeom
 
-        model = SkyEllipse("2 deg", "2 deg", "1 deg", 0.8, "30 deg")
+        model = SkyEllipse("2 deg", "2 deg", "1 deg", 0.8, "20 deg", frame='galactic')
```
Comment (lmohrmann) [inline gammapy/image/models/core.py]: I guess you want to put `"galactic"` rather than `'galactic'`, to be compliant with black.

**Comment by lmohrmann** [comment]
> Maybe make a https://astropy-regions.readthedocs.io/en/latest/api/regions.EllipseSkyRegion.html and overplot it on the model image. Is the rotation angle definition the same?

Seems that astropy defines the angle w.r.t. the "width axis", which is only well defined before the rotation. So I guess the definitions are not the same, but we cannot really follow the astropy definition with our current parametrisation, and I also don't see why we should.

**Comment by cdeil** [comment]
@lmohrmann - Did you try overplotting the ellipse (which has a SkyCoord with the same frame as your model)? @luca-giunti - maybe you could try and share an updated notebook with this added?

I would be surprised if it doesn't come out to be the same:
https://astropy-regions.readthedocs.io/en/latest/_modules/regions/shapes/ellipse.html#EllipseSkyRegion.to_pixel

It is a concern if the ellipse regions we use have different angles than the models we use in Gammapy.

**Comment by luca-giunti** [comment]
@cdeil I tested in this [notebook](https://github.com/luca-giunti/share/blob/master/Untitled15.ipynb) the consistency with the `regions.EllipseSkyRegion` convention.

I think it is basically the same, so this should not be a problem. @lmohrmann I will add a short test for the rotation, thanks for your review! 

I also see there is some travis fail, but I think those are unrelated with this PR

**Comment by cdeil** [comment]
@luca-giunti - Great!

File: gammapy/image/models/core.py [hunk 2]
```diff
@@ -332,15 +332,8 @@ class SkyEllipse(SkySpatialModel):
         skymap = Map.from_geom(m_geom, data=vals.value)
 
         _, ax, _ = skymap.smooth("0.05 deg").plot()
-        transform = ax.get_transform('galactic')
```
Comment (adonath) [inline gammapy/image/models/core.py]: I think this illustration was useful...maybe adapt and keep?

**Comment by adonath** [comment]
@luca-giunti Is this PR ready to be merged? I've left one minor inline comment, do you have time to address it?

**Comment by luca-giunti** [comment]
@adonath yes, I'll do it now

**Comment by luca-giunti** [comment]
Here is the new figure produced in the example
new_ellipse

**Pull request merged**